tests/nodeos_extra_packed_data_test.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
+
+from testUtils import Account
+from testUtils import Utils
+from Cluster import Cluster, PFSetupPolicy
+from WalletMgr import WalletMgr
+from Node import Node
+from Node import ReturnType
+from TestHelper import TestHelper
+from TestHelper import AppArgs
+
+import decimal
+import re
+import json
+import os
+import copy
+import math
+import time
+
+###############################################################
+# nodeos_run_test
+#
+# General test that tests a wide range of general use actions around nodeos and keosd
+#
+###############################################################
```

Comment:
Not sure why this test is included in this PR; seems unrelated.
This test should be cleaned up to remove all the cruft that is not used.

Reply:
yes, was added due to automatic cherry-pick, will remove